tests/unit/storage/data/external/GcpService.spec.js

```diff
@@ -86,33 +160,68 @@ const operations = [
     },
 ];
 
+async function cleanupServer(httpServer, sockets) {
+    if (httpServer) {
+        sockets.forEach(socket => {
+            if (!socket.destroyed) {
+                socket.destroy();
+            }
+        });
+
+        await new Promise((resolve) => {
+            const timeout = setTimeout(() => {
+                resolve();
+            }, 3000);
+
+            httpServer.close(() => {
+                clearTimeout(timeout);
+                resolve();
+            });
+        });
+    }
+}
+
 describe('GcpService request behavior', () => {
     jest.setTimeout(120000);
     let httpServer;
     let client;
+    let sockets = [];
 
     beforeAll(done => {
         client = new GCP({
-            endpoint: `http://${host}`,
-            maxRetries: 0,
-            s3ForcePathStyle: false,
-            accessKeyId,
-            secretAccessKey,
+            s3Params: {
+                endpoint: `http://${host}`,
+                maxAttempts: 1,
+                forcePathStyle: false,
+                region: 'us-east-1',
+                credentials: {
+                    accessKeyId: accessKeyId,
+                    secretAccessKey: secretAccessKey,
+                },
```

Comment:
suprises the linter did not see this (it does in my IDE):
```suggestion
                    accessKeyId,
                    secretAccessKey,
                },
```

same a few times below